src/templates/_changelog.hbs

```diff
@@ -0,0 +1,8 @@
+<div class="changelog-container js-changelogContainer nice-scrollbar">
+    <h1 class="changelog-hd">Changelog</h1>
+    <h2 class="changelog-version">v{{version}}</h2>
+    <p class="js-changelog blackBox"></p>
+    <div class="changelog-view-full">
+        <a href="https://github.com/openscope/openscope/releases">View full changelog</a>
```

Comment:
IMHO the link should open in a new window/tab.

Reply:
We are pushing that back to a new issue for some reason: #1322.